docs/code/oi_keymap.rst

```diff
@@ -0,0 +1,127 @@
+=========================
+Operator Interface (OI)
+=========================
+
+The `OI.py` and `keymap.py` modules in the `7407-DriveCode-Reefscape` repository collaboratively manage the operator interface, facilitating the mapping of controller inputs to specific robot commands. This structured approach ensures intuitive and efficient control during robot operation.
+
+Keymap Module
```

Comment:
This is a clear discussion of how the OI worked very specifically last year, but I am hopeful that you can look at this and create a sense of how we use the OI.py file generally. What is the philosophy? What kinds of code do people put here, and why?